docs-ref-conceptual/query-azure-cli.md

```diff
@@ -140,7 +140,7 @@ Querying Boolean values is slightly different.  There are two options:
 az account list --query "[?isDefault]"
 
 # If you want a false value, use an escape character.
-az account list --query "[?isDefault == ``false``]"
+az account list --query "[?isDefault == ``false`]"
```

Comment:
This is not correct.  There is an extra backtick in front of `false`.  Please change to -->

![image](https://user-images.githubusercontent.com/58762114/162222547-2681e595-88c5-42ef-9911-ad0e552c5b9a.png)